help/help.c

```diff
@@ -629,6 +632,8 @@ static int help_read_dir(const char *path, struct Vector *DocList)
     char *path2 = vector_get(file_paths, i, NULL);
     help_doc_gather(&DocList, path2);
   }
+  vector_free(&file_paths, NULL);
```

Comment:
We're done with these paths, so free them.
See also, the change to `scan_dir()`.